config.py

```diff
@@ -1969,3 +2018,430 @@ def annotations_to_events(
+                                    session=session))
+        raw.load_data()
+        for ch_name, (anode, cathode) in eeg_bipolar_channels.items():
+            msg = f'    {anode} – {cathode} -> {ch_name}'
+            logger.info(gen_log_message(message=msg, step=0, subject=subject,
+                                        session=session))
+            mne.set_bipolar_reference(raw, anode=anode, cathode=cathode,
+                                      ch_name=ch_name, drop_refs=False,
+                                      copy=False)
+        # If we created a new bipolar channel that the user wishes to
+        # # use as an EOG channel, it is probably a good idea to set its
+        # channel type to 'eog'. Bipolar channels, by default, don't have a
+        # location, so one might get unexpected results otherwise, as the
+        # channel would influence e.g. in GFP calculations, but not appear on
+        # topographic maps.
+        if (eog_channels and
+                any([eog_ch_name in eeg_bipolar_channels
+                     for eog_ch_name in eog_channels])):
+            msg = 'Setting channel type of new bipolar EOG channel(s) …'
+            logger.info(gen_log_message(message=msg, step=0, subject=subject,
+                                        session=session))
+        for eog_ch_name in eog_channels:
+            if eog_ch_name in eeg_bipolar_channels:
+                msg = f'    {eog_ch_name} -> eog'
+                logger.info(gen_log_message(message=msg, step=0,
+                                            subject=subject,
+                                            session=session))
+                raw.set_channel_types({eog_ch_name: 'eog'})
+
+
+def _set_eeg_montage(raw, subject, session) -> None:
+    """Set an EEG template montage if requested.
+
+    Modifies ``raw`` in-place.
+    """
+    montage_name = eeg_template_montage
+    if get_datatype() == 'eeg' and montage_name:
+        msg = (f'Setting EEG channel locations to template montage: '
+               f'{montage_name}.')
+        logger.info(gen_log_message(message=msg, step=0, subject=subject,
+                                    session=session))
+        montage = mne.channels.make_standard_montage(montage_name)
+        raw.set_montage(montage, match_case=False, on_missing='warn')
+
+
+def _fix_stim_artifact_func(raw: mne.io.BaseRaw) -> None:
+    """Fix stimulation artifact in the data."""
+    if not fix_stim_artifact:
+        return
+
+    events, _ = mne.events_from_annotations(raw)
+    mne.preprocessing.fix_stim_artifact(
+        raw, events=events, event_id=None,
+        tmin=stim_artifact_tmin,
+        tmax=stim_artifact_tmax,
+        mode='linear'
+    )
+
+
+def import_experimental_data(
+    *,
+    subject: str,
+    session: Optional[str] = None,
+    run: Optional[str] = None,
+    save: bool = False
+) -> mne.io.BaseRaw:
+    """Run the data import.
+
+    Parameters
+    ----------
+    subject
```

Comment:
```suggestion
    subject : str
```
see also below

Reply:
With type-annotated functions I always omit the type in the docstring because it's an unnecessary redundancy. IDEs will display the expected type and the doc build system extracts them automatically, too